Sources/StreamChat/APIClient/Endpoints/Payloads/PollsPayloads.swift

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct PollPayloadResponse: Decodable {
+    let duration: String
+    let poll: PollPayload
+
+    init(duration: String, poll: PollPayload) {
+        self.duration = duration
+        self.poll = poll
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case duration
+        case poll
+    }
+}
+
+struct QueryPollsPayloadResponse: Decodable {
```

Comment:
nit: Shouldn't this be PollsList? 🤔 
```suggestion
struct PollsListPayloadResponse: Decodable {
```

Reply:
done